src/components/subscriptionDialog.tsx

```diff
@@ -147,6 +148,7 @@ export const SubscriptionDialog = ({
             <DialogTemplate<SubscriptionFormikValues>
                 advancedFields={advancedFields}
                 basicFields={basicFields}
+                schemaInputField={schemaInputField}
```

Comment:
It looks like, we broke the[ interface segregation principle](https://en.wikipedia.org/wiki/Interface_segregation_principle) and  `no code should be forced to depend on methods it does not use`.  It's not an ideal solution, but let's make schemaInputField optional.